src/mds/MDSRank.cc

```diff
@@ -2734,7 +2734,12 @@ void MDSRankDispatcher::handle_asok_command(
     r = config_client(client_id, !got_value, option, value, *css);
   } else if (command == "scrub start" ||
 	     command == "scrub_start") {
-    if (whoami != 0) {
+    if (is_any_replay()) {
+      *css << "MDS is replaying log";
+      r = -CEPHFS_EAGAIN;
+      goto out;
+    }
```

Comment:
```suggestion
    if (!is_active)) {
      *css << "MDS is not active";
      r = -CEPHFS_EINVAL;
      goto out;
    }
```

Reply:
The tracker mentions to prevent scrub in case of standby-replay.By this, I should assume that we do not allow scrub command for all the states other than ACTIVE, right?

Reply:
That is right - there is no point in running scrub for MDSs that do not hold any rank.